src/shared/glemu.h

```diff
@@ -44,7 +44,7 @@ namespace gle
         static inline void name##f(float x) { glVertexAttrib1##f##_(index, x); } \
         static inline void name##f(float x, float y) { glVertexAttrib2##f##_(index, x, y); } \
         static inline void name##f(float x, float y, float z) { glVertexAttrib3##f##_(index, x, y, z); } \
-        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4##f##_(index, x, y, z, w); } \
+        static inline void name##f(float x, float y, float z, float w) { glVertexAttrib4f_(index, x, y, z, w); } \
```

Comment:
It was not only this line; the ones above have the same thing.

Reply:
well, turns out that most of that code is *plain useless* so I'm spending some extra time remove unneeded stuff.